frontend/package.json

```diff
@@ -23,6 +23,7 @@
     "axios": "^1.4.0",
     "prop-types": "^15.8.1",
     "react": "^18.2.0",
+    "react-device-detect": "^2.2.3",
```

Comment:
No need for this, there is a prebuilt custom hook `useScreen` for this. You can use that hook.

Reply:
It should only be visible in `lg` and `xl`.
Here is the definition of the hook for reference:
https://github.com/bsoc-bitbyte/busify/blob/49e4446bc6aa55ddd428c05ac40601e4182cc542/frontend/src/customHooks/useScreen.tsx#L4